frontend/front/src/pages/Admin/home/HomeTable.js

```diff
@@ -76,7 +88,20 @@ const HomeTable = () => {
     {
       field: "completed",
       headerName: "Completed",
-      renderCell: (params) => (params.row.completed === true ? "Yes ✅" : "No"),
+      renderCell: (params) =>
+        params.row.completed === true ? (
```

Comment:
This can probably just be `params.row.completed` actually - but that wasn't your addition so whatever

Reply:
I changed it.